Makefile

```diff
@@ -115,6 +115,7 @@ $(BUILD)/$(BOARD).txt: $(ROM)
 	( \
 	cd $(BUILD) ; \
 	$(pwd)/bin/extract-firmware \
+		$(EXFW_OPT) \
```

Comment:
Same here

Reply:
Here we intend to add "--repack" parameter for extract-firmware, because the FVs of AMI BIOS are compressed by LZMA format, but LZMA decompress protocol is not built by default in the EDK2 code, so adding "--repack" parameter to decompress the FVs, otherwise the Linuxboot BIOS can`t boot up on TiogaPass.
@morganjangwiwynn